vendor/osp16.2/apply_arbiterd_patch.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
so i have configured pre-commit to check the executable permission on scripts.

can you chmod +x this file so that it passes the check.
that is why its currently failing ci